rust_src/src/marker.rs

```diff
@@ -161,4 +171,56 @@ pub fn copy_marker(marker: LispObject, itype: LispObject) -> LispObject {
     new
 }
 
+/// Return t if there are markers pointing at POSITION in the current buffer.
+#[lisp_fn]
+pub fn buffer_has_markers_at(position: LispObject) -> LispObject {
```

Comment:
`position` is a EmacsInt so you can declare that here instead of in the function. And return the `bool`